property_tests/tests/canister_methods/http_request/package.json

```diff
@@ -0,0 +1,12 @@
+{
+    "scripts": {
+        "test": "ts-node --transpile-only --ignore=false test/test.ts"
```

Comment:
this seems fishy, no prepare step?

Reply:
none of the property tests have the pretests